### PR TITLE
Use absolute import for bootstrap-datepicker

### PR DESCRIPTION
**Pull request by tnagorra**
- Remove background-image for geo-picker

### CODE REVIEW

File: src/widget/geo/geopicker.scss [hunk 1]
```diff
@@ -1116,13 +1116,13 @@ $input-button-border: 2px solid #aaaaaa;
 }
 
 .leaflet-control-layers-toggle {
-    background-image: url(images/layers.png);
+    // background-image: url(images/layers.png);
```
Comment (tnagorra) [inline src/widget/geo/geopicker.scss]: I could not find the image `layers.png` anywhere in the repository.

**Comment by MartijnR** [comment]
Thank you. Would you perhaps like to do the same for the other image file references in that stylesheet?

**Comment by tnagorra** [comment]
@MartijnR the only other reference to `url()` in that stylesheet I could find is
https://github.com/the-deep/enketo-core/blob/4cb8558ffbc4b44f1258f0280fd68a7d149505eb/src/widget/geo/geopicker.scss#L843-L847

Is it safe to comment this out?

**Comment by MartijnR** [comment]
Oops, sorry. Turns out I didn't review your PR properly and only looked a the commented change above.

**Comment by tnagorra** [comment]
I am using webpack to build the scss and the absolute import is provided by webpack.
I didn't know scss was built using grunt in this project. Will have to look into grunt and grunt-sass to get absolute import working.

**Comment by MartijnR** [comment]
I've merged it, but reverted the import. It would still be welcome, but I don't have time myself to dive into SCSS processors atm and I believe our current one may be faster than the competition, which is quite cool during development. Thanks!

**Pull request closed without merging**